packages/storage-events/src/event.spec.js

```diff
@@ -42,6 +42,13 @@ describe("Event", () => {
     });
   });
 
+  it("can set a status without crashing", done => {
+    Event.create({ eventType: "ADDED_TODO", data: { status: "done" } }).then(event => {
```

Comment:
We could return or await this promise to make the tests a bit easier to understand than using the `done()` callback (can be done as a separate PR as we already did it like this in this file)